packages/k2-alpine/src/theme/tokens/text.ts

```diff
@@ -0,0 +1,154 @@
+// K2 Foundation - Typography
+// https://www.figma.com/file/hDSl4OUgXorDAtqPZtCUhB/K2-Foundation?type=design&node-id=3390-22100
+
```

Comment:
can you create storybook's stories for the all the texts and colors? similar to the ones in the current k2

Reply:
Yes that was in my next steps.

Reply:
do you mean in a new pr or within this pr?

Reply:
New PR. It'll be done with the `Typography` component.